rafs/src/metadata/mod.rs

```diff
@@ -383,6 +385,13 @@ impl RafsSuperConfig {
             )));
         }
 
+        if (self.batch_size == 0) != (meta.batch_size == 0) {
```

Comment:
I still think this check is unnecessary.

Reply:
Removed.